server/actions/compute-handler.js

```diff
@@ -19,9 +19,9 @@ function computeHandler(req, res) {
     const startTime = new Date();
     compute(user, ship, { preview: true })
     .then(result => {
-      const logs = result.logs;
-      if (logs && logs.length) {
-        logs.map(line => req.hull.client.logger.debug("preview.console.log", line));
+      const logsForLogger = result.logsForLogger;
+      if (logsForLogger && logsForLogger.length) {
+        logsForLogger.map(line => req.hull.client.logger.debug("preview.console.log", line));
```

Comment:
@mkawalek this is the place which logs log lines when running a preview. We have also a logger in `user-update.js` which does the same but for notification handling. We need to adjust it too.